docs/config.json

```diff
@@ -2,16 +2,16 @@
   "navigation": [
     {
       "icon": "home",
-      "title": "Home",
+      "title": "Get Started",
       "entries": [
         {
           "title": "Introduction to Teleport",
           "slug": "/"
         },
-        {
-          "title": "Documentation Overview",
-          "slug": "/documentation-overview/"
-        },
+	{
```

Comment:
Looks like you've changed spaces to tabs.

Not a big deal, as this file gets overwritten by post-release workflows all the time, but it does introduce superfluous diffs.